### PR TITLE
Fix syntax errors

### PR DESCRIPTION
**Pull request by HenItzhaki**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [x] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Comment by HenItzhaki** [comment]
Signed-off-by: Hen Itzhaki <itzhakihen@gmail.com>

File: 3.0/windows/2019/Dockerfile [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
-  Write-Output 'Cleaning up installation'; `
-  Remove-Item c:\rubyinstaller-devkit-3.0.0-1-x64.exe -Force; `
+RUN powershell -Command \
+  $ErrorActionPreference = 'Stop'; \
+  Write-Output 'Downloading Ruby + DevKit'; \
+  [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+  (New-Object System.Net.WebClient).DownloadFile('https://github.com/oneclick/rubyinstaller2/releases/download/RubyInstaller-3.0.0-1/rubyinstaller-devkit-3.0.0-1-x64.exe', 'c:\\rubyinstaller-devkit-3.0.0-1-x64.exe'); \
+  Write-Output 'Installing Ruby + DevKit'; \
+  Start-Process c:\rubyinstaller-devkit-3.0.0-1-x64.exe -ArgumentList '/verysilent /dir=C:\\ruby27' -Wait ; \
+  Write-Output 'Cleaning up installation'; `\
```
Comment (tas50) [inline 3.0/windows/2019/Dockerfile:27]: You probably want to remove the ` here

**Comment by tas50** [comment]
@HenItzhaki Thanks for the PR. Before we can merge this you'll need to sign the commit off for the DCO. https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco

File: 3.0/windows/2019/Dockerfile (hunk 1 above)
Comment (HenItzhaki) [reply 3.0/windows/2019/Dockerfile:27]: Changed from ` to \

**Comment by tas50** [comment]
@christopher-snapp This one is for you to review

**Comment by christopher-snapp** [comment]
@HenItzhaki Thanks very much!

The changes to the escape character look good for 3.0.  To be consistent, I'd like to get those changes made to the 2.6 and 2.7 windows 2019 docker files as well.

The DCO requires a sign-off message to appear on each commit in the pull request and is usually just appended to your commit message rather than as its own separate commit message.  Feel free to squash your commits with the DCO tacked on the end.

**Comment by christopher-snapp** [comment]
@HenItzhaki this PR is looking good for the changes you're making, the only thing holding up the merge is the DCO check.

note that of the four commit messages you've got the first is still missing a DCO, the second is correct, and the third and fourth are missing the angle brackets around the email which causes the DCO check to fail

my recommendation would be to rebase and squash your commits into a single commit with this message:

```
Fix Windows escape characters

Signed-off-by: Hen Itzhaki <itzhakihen@gmail.com>
```

**Review by HenItzhaki — COMMENTED** [review]
Signed-off-by: Hen Itzhaki <itzhakihen@gmail.com>

**Pull request merged**